rest_api/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import patterns, include, url
+from django.contrib import admin
+
+
+urlpatterns = patterns('',
+    url(r'^grappelli/', include('grappelli.urls')),
+    url(r'^admin/', include(admin.site.urls)),
+    url(r'^author/', include('author_api.urls')),
+    # lets us access endpoints through /post or /author
+    url(r'^posts/', include('content_api.urls')),
+    url(r'^author/', include('content_api.urls')),
```

Comment:
Added this set of routes as well since we need to support /author/:id/posts. That doesn't mean I got it working.